profession/get-help-with-architecture.md

```diff
@@ -0,0 +1,47 @@
+---
+category: profession
+expires: 2023-02-15
+---
+
+```mermaid
+sequenceDiagram
+
+  title Get help with architecture in DfE
+
+  participant Requestor
+  participant DDaT Business Partner
+  participant Portfolio
+  participant Roadmap
+  participant Emergency Triage
+  participant Profession
+
+  Requestor->DDaT Business Partner:Define the need and raise
+  note over DDaT Business Partner: Is this definitely an \narchitecture need?
```

Comment:
Is this a typo....\narchitecture.....?

Reply:
\n is a line break in the mermaid sequence diagram code, these are required